AltUseVendors/__init__.py

```diff
@@ -183,6 +207,7 @@ def refill_ammo(PC: WillowPlayerController, vendor: WillowVendingMachine) -> Non
     """
     for _, _, pool in iter_ammo_data(PC, vendor):
         pool.SetCurrentValue(pool.GetMaxValue())
+    PC.pawn.PlayAkEvent(unrealsdk.FindObject("AkEvent", AKE_BUY))
```

Comment:
Alt use vendors has a bit of an undeserved reputation for causing stutters, despite having been optimized several times. Find object is a somewhat expensive call, so I want to be careful about adding it. It may well turn out to be the best solution, but there's some things I want checked first.

Firstly, I assume these objects are map specific, so we can't do a root level find object? And that they depend on other assets so we can't construct a new version like we do interaction icons? If we can get a single global reference, prefer that.

Now if that's not possible (which I'm pretty sure it isn't), can we use keep alive. On first use, do use a find object, but then call keep alive, and keep the direct object reference for all future calls.

Regardless of what solution ends up best, I also want to see some handling for if find object returns None. If it's a map specific object, there's always a chance of something weird happening on some map you didn't test, don't want to cause errors unwittingly.

Reply:
I don't 100% understand how loading objects and storing references to them works, but using FindObject outside the class or on mod Enable returns None.
Added `find_and_play_akevent` method to handle storing and retrieving them when called.